src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -261,12 +261,7 @@ private async Task<bool> IsIntegratedConsoleEnabledAsync()
                 }
             }
 
-            string? commandLineArgs = resolvedProfile.CommandLineArgs?
-                .Replace("\r\n", " ")
-                .Replace("\r", " ")
-                .Replace("\n", " ")
-                .Replace("\\r\\n", "\r\n")
-                .Replace("\\n", "\n");
+            string? commandLineArgs = resolvedProfile.CommandLineArgs?.Replace("\r\n", " ");
```

Comment:
As suggested in https://github.com/dotnet/project-system/pull/7919#discussion_r806293499 I'd consider using this instead:

```suggestion
            string? commandLineArgs = resolvedProfile.CommandLineArgs is null
                ? null
                : Regex.Replace(resolvedProfile.CommandLineArgs, "[\r\n]+", " ");
```

There is no guarantee that the file doesn't contain other kinds of line endings. Git can meddle with line endings, for example.

The bug in 17.2 was caused by the addition of the last two lines. That was escaping the literal `@"\n"` (i.e. two characters) rather than the newline character.